### PR TITLE
Use `.hasField('…')` to check for existence of fields

### LINKED ISSUE #343: Twig breaks on `is defined`
This should work: 

```twig
    {% if record.foo is defined %}
        It should be defined if we get here
        {{ dump(record.foo) }}
    {% endif %}
```

Instead I get this error: 

screenshot 2019-03-02 at 15 15 20

### PR DESCRIPTION
**Pull request by bobdenotter**
Fixes #343 

This PR can serve as a workaround, allowing us to carefully make an elegant solution in #344

### CODE REVIEW

File: public/theme/skeleton/record.twig [hunk 1]
```diff
@@ -2,18 +2,21 @@
 
 {% block main %}
 
-    <h1>{{ record.title }}</h1>
+    {% if record.hasField('title') %}
```
[the remark this answers is not in the document]
Comment (bobdenotter) [reply public/theme/skeleton/record.twig]: Yeah, the example is a bit convoluted, but I needed a "hook" for in the e2e test.

I think we eventually will have a 'theme' that's used in testing / development, that's separate from the real "default" theme, which will be more applicable for real world use.

File: public/theme/skeleton/partials/_master.twig [hunk 2]
```diff
@@ -7,9 +7,9 @@
         {# make sure we always display a proper title: The record's title if there is one, appended with the
            sitename. If there is no title, we append the sitename with the payoff, if there is one. #}
         <title>
-            {%- if record.title is defined %}{{ record.title|striptags }} | {% endif -%}
+            {%- if record.hasField('title') %}{{ record.title|striptags }} | {% endif -%}
                 {{ config.get('general/sitename') -}}
-            {% if record.title is not defined and config.get('general/payoff') %} | {{ config.get('general/payoff') }}{% endif -%}
+            {% if not record.hasField('title') and config.get('general/payoff') %} | {{ config.get('general/payoff') }}{% endif -%}
```
[the remark this answers is not in the document]
Comment (bobdenotter) [reply public/theme/skeleton/partials/_master.twig]: Fixed.

File: public/theme/skeleton/record.twig (hunk 1 above)
[the remark this answers is not in the document]
Comment (JarJak) [reply public/theme/skeleton/record.twig]: Hmm and then we have 2 themes to support.
I would vote for having 1 theme and tests for it.

[the remark this answers is not in the document]
Comment (bobdenotter) [reply public/theme/skeleton/record.twig]: We'll see when we get to that point! It never was much of a hassle to maintain both `skeleton` and `base-2018`, so i'm not too worried.  😇

**Pull request merged**